config/ytt/ako-operator.yaml

```diff
@@ -42,13 +42,26 @@ spec:
             - --metrics-addr=127.0.0.1:8080
           command:
             - /manager
-          image: #@ dockerImage()
+          image: gwang550/akoo:test
```

Comment:
Change this back after the test

Reply:
Thanks for catching that. The believe the [api/v1alpha1/zz_generated.deepcopy.go](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/pull/66/files#diff-d900babf741981553f1ae363bc2d0062aab6a95256dfa091b688df589e9a5c5d) was formatted by IDE. The only file generated is webhook file. The test was added by me later.